packages/contracts-bedrock/package.json

```diff
@@ -21,7 +21,7 @@
     "test": "pnpm build:differential && pnpm build:fuzz && forge test",
     "coverage": "pnpm build:differential && pnpm build:fuzz && forge coverage",
     "coverage:lcov": "pnpm build:differential && pnpm build:fuzz && forge coverage --report lcov",
-    "gas-snapshot:no-build": "forge snapshot --no-match-test 'testDiff|testFuzz|invariant|generateArtifact'",
+    "gas-snapshot:no-build": "forge snapshot --match-contract GasBenchMark",
```

Comment:
This will catch any contract with the string `GasBenchMark` in it, enabling benchmarks to be written in any test file.